richie/settings.py

```diff
@@ -256,14 +257,36 @@ class Base(DRFMixin, ElasticSearchMixin, Configuration):
     CMS_PERMISSION = True
 
     CMS_PLACEHOLDER_CONF = {
-        "courses/cms/course_detail.html course_video": {
+        # Course detail
+        "courses/cms/course_detail.html course_teaser": {
             "name": _("Course teaser"),
             "plugins": ["VideoPlayerPlugin"],
             "limits": {"VideoPlayerPlugin": 1},
         },
-        "courses/cms/course_detail.html course_syllabus": {  # all plugins allowed
-            "name": _("Course Syllabus")
+        "courses/cms/course_detail.html course_syllabus": {
+            "name": _("Course Syllabus"),
+            # Enforce Section plugin usage at first level
+            "plugins": ["SectionPlugin"],
         },
+        "courses/cms/course_detail.html course_format": {
+            "name": _("Format"),
+            "plugins": ["TextPlugin"],
+        },
+        "courses/cms/course_detail.html course_prerequisites": {
+            "name": _("Prerequisites"),
+            "plugins": ["TextPlugin"],
+        },
+        "courses/cms/course_detail.html course_team": {"name": _("Course team")},
+        "courses/cms/course_detail.html course_plan": {
+            "name": _("Course plan"),
+            "plugins": ["SectionPlugin"],
```

Comment:
I think we don't need the section plugin here but a specific plugin to do course plans... separate PR!

Reply:
Could you open an issue about this new plugin to create ?